data/PL.poland/validate_tabular_remote_source.py

```diff
@@ -0,0 +1,71 @@
+""" A processor to download a data-file from a link and validate it."""
```

Comment:
I guess that this is still a work in progress?

What is the purpose of this code?